packages/sdk-js/src/GrowthBook.ts

```diff
@@ -488,12 +500,37 @@ export class GrowthBook<
 
     // Apply new changes
     if (result.inExperiment) {
-      const undo = this._applyDOMChanges(result.value);
-      if (undo) {
-        this._activeAutoExperiments.set(experiment, {
-          undo,
-          valueHash,
-        });
+      if (result.value.urlRedirect) {
+        const url = experiment.persistQueryString
+          ? result.value.urlRedirect + window.location.search
+          : result.value.urlRedirect;
```

Comment:
This wont work. Need to merge curr page query params with any possible supplied query params in the redirect URL

Reply:
Use url in the context to grab query params, not window.location.search